src/test/resources/artifacts/AXIS2/config/test_axis2_server_9000.xml

```diff
@@ -0,0 +1,378 @@
+<!--
```

Comment:
ok i will change it

On Tue, Jun 28, 2016 at 11:19 AM, Rajjaz Mohammed <notifications@github.com>
wrote:

> In src/test/resources/artifacts/AXIS2/config/test_axis2_server_9000.xml
> <https://github.com/wso2-extensions/esb-connector-asana/pull/1#discussion_r68700598>
> :
>
> > @@ -0,0 +1,378 @@
> > +<!--
>
> Wrong license header/template. Please use WSO2 template
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/wso2-extensions/esb-connector-asana/pull/1/files/c177472706923e17a306dcf3ca6ca20eac47a8e7#r68700598>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AMLf3QsDrQDiMP2jy9LowEBOoNQEDaJHks5qQLXQgaJpZM4I-19F>
> .
>